way_to_home/utils/vehicle_data.py

```diff
@@ -0,0 +1,32 @@
+"""
+    This module use for parsing data got in real time from 'Easy Way' source
+    in GTFS format.
+"""
+
+import requests
+from google import protobuf
+from google.transit import gtfs_realtime_pb2
+
+
+FEED = gtfs_realtime_pb2.FeedMessage()
+RESPONSE = requests.get('http://track.ua-gis.com/gtfs/lviv/vehicle_position')
+FEED.ParseFromString(RESPONSE.content)
+
+
+def get_vehicle_data(vehicle_id):
+    """
+        This function takes certain vehicle id as parameter and
+        returns appropriate data about it's location, speed etc.
+    """
+    vehicle_data_json = {}
+    for entity in FEED.entity:
+        if entity.vehicle.vehicle.id == vehicle_id:
```

Comment:
Is it possible that some data will be missing in response? For example `entity.vehicle.position.longitude`

Reply:
Yes, it's possible when a vehicle is not on the trip and it's data not tracked. In that case, the function will return an empty dictionary

Reply:
```suggestion
    vehicle_data_json['route_id'] = entity.vehicle.trip.route_id
    vehicle_data_json['lat'] = entity.vehicle.position.latitude
    vehicle_data_json['lon'] = entity.vehicle.position.longitude
    vehicle_data_json['speed'] = entity.vehicle.position.speed
```
I think that you need to put it into the block `try except`, because if some data is missed, you will get an error.